aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts

```diff
@@ -41,6 +47,6 @@ export interface IAttributesRef {
   updateScales?: (xScale: IGetAxisScale, yScale: IGetAxisScale) => void;
   setActiveLine?: (lineKey: string) => void;
   updateHoverAttributes?: (xValue: number) => void;
-  updateFocusedChart?: (mousePos: [number, number]) => IActivePoint;
+  updateFocusedChart?: (props?: IUpdateFocusedChartProps) => void;
```

Comment:
lets write function arguments as params instead of props

Reply:
fine i will fix this , thanks

Reply:
for only simple functions we decide to do yes ? for react components we can use props as well